### PR TITLE
Use Console.ResetColor() to restore color settings

### PR DESCRIPTION
**Pull request by jkotas**
Reading current foreground color is not supported on Unix

### CODE REVIEW

**Comment by jkotas** [comment]
@FiveTimesTheFun @stephentoub  Could you please take a look?

File: src/xunit.console.netcore/Program.cs [hunk 1]
```diff
@@ -86,7 +88,9 @@ public static int Main(string[] args)
             }
             finally
             {
+#if !NETCORE
```
Comment (stephentoub) [inline src/xunit.console.netcore/Program.cs]: Consider calling Console.ResetColor() here instead; that'll have the same effect, and is supported.

**Comment by stephentoub** [comment]
One comment, otherwise LGTM.

**Comment by FiveTimesTheFun** [comment]
Thank you - LGTM too.

**Pull request merged**